scripts/run-android.sh

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env bash
+set -euo pipefail
+set -m # needed to access jobs
+
+export BUILD_ENV=debug
+export BUILD_TYPE=debug
+GIT_ROOT=$(cd "${BASH_SOURCE%/*}" && git rev-parse --show-toplevel)
+
+# generates android debug build when we pass the debug flag
+"${GIT_ROOT}"/scripts/build-android.sh
+
+# installs the resulting apk on running emulator or android device
+adb install ./result/app-debug.apk
+
+# start metro and go to background
+nohup "${GIT_ROOT}"/scripts/start-react-native.sh 2>&1 | tee metro-server-logs.log &
+
+echo "Waiting for Metro server to be ready..."
+while ! grep -q "Welcome to Metro" metro-server-logs.log; do
+  echo -n "."
+  sleep 1
+done
+
+# start the installed app
+adb shell monkey -p im.status.ethereum.debug 1
+
+METRO_JOB=$(jobs | grep "metro")
+if [ $? -eq 0 ]; then
+    METRO_ID=$(echo "$METRO_JOB" | awk '{print $1}' | tr -d '[]+' )
+else
+    echo "Metro server background information not found."
```

Comment:
I guess we should fail the script here.

Reply:
hmm yes we should, this echo would not be helpful and the script would wait forever.

Reply:
addressed here : https://github.com/status-im/status-mobile/pull/18784/commits/1920a92d166733e43f97c04a41ae81bad1bd8926